contributing/README.md

```diff
@@ -0,0 +1,23 @@
+# Contributing
+
+Before you start contributing to the project, you should familiarize yourself with the [documentation](vmware.github.io/vmware-aria-operations-integration-sdk/),
+[issue tracker](https://github.com/vmware/vmware-aria-operations-integration-sdk/issues), [project structure](index.md),
+and basic functioning of every component.
```

Comment:
Rather than `every component`, it would be better to provide a specific list of pages someone should review.